package.json

```diff
@@ -10,17 +10,20 @@
     "lint": "tslint 'src/**/*.ts{,x}'",
     "lint-fix": "tslint --fix 'src/**/*.ts{,x}'",
     "test": "jest --coverage --coverageReporters=cobertura",
+    "test-coverage-html": "jest --coverage --coverageReporters=html",
     "test-watch": "jest --watchAll"
   },
   "devDependencies": {
     "@types/cli-table": "^0.3.0",
+    "@types/dotenv": "^6.1.1",
     "@types/jest": "^24.0.18",
     "@types/js-yaml": "^3.12.1",
+    "@types/node": "^12.7.8",
     "@types/mock-fs": "^3.6.30",
-    "@types/node": "^12.7.4",
     "@types/node-emoji": "^1.8.1",
     "@types/shelljs": "^0.8.5",
     "@types/uuid": "^3.4.5",
+    "dotenv": "^8.1.0",
```

Comment:
`dotenv` should be in `dependencies`

Reply:
yes. Initially not planned to take a dependency, but after discussing with @evanlouie, decided to add under dependencies. Good catch.

Reply:
Ok, no worries i'll move it to dependencies in my PR